app/routes/circulars.edit.$circularId/CircularEditForm.tsx

```diff
@@ -164,150 +181,214 @@ export function CircularEditForm({
   const changesHaveBeenMade =
     body.trim() !== defaultBody.trim() ||
     subject.trim() !== defaultSubject.trim() ||
-    format !== defaultFormat
+    format !== defaultFormat ||
+    submitter !== updatedSubmitter?.trim() ||
+    Date.parse(`${defaultDate[0]} ${defaultDate[1].substring(0, 5)}`) !==
```

Comment:
You're doing a lot of unnecessary conversion back and forth. I suggest instead:

```ts
export function CircularEditForm({
  ...
 defaultCreatedOn,
 ...
} : {
  defaultCreatedOn
})
{
  const [createdOn, setCreatedOn] = useState(defaultCreatedOn)
  ...
  const changesHaveBeenMade = ... || createdOn !== defaultCreatedOn
}
```

Reply:
I originally went for something like this, but the default CreatedOn time have accuracy as HH:MM:SS, where these input are only allowing HH:MM, so it becomes a bit wonky